### DISCUSSION TITLE
Support for DE0 CV boards

### DISCUSSION THREAD

**Discussion by rafa-rrayes** [FPGAwars/icestudio#813] [New Boards Requests]
Hey there! are you working on support for this board? How can I help developing support?

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
